Developers/Writing_Bears.rst

```diff
@@ -150,6 +150,42 @@ This will show the user a bunch of information related to the bear like:
 - A description of what the bear does - The sections which uses it - The
 settings it uses (optional and required)
 
+.. note::
+
+    The bears are not yet installed. We still have to specify
+    the bear directory using ``-d`` or ``--bear-dirs`` flag.
+
+
+Installing the New coala-bears Locally
```

Comment:
The issue was to describe how to install locally written bear it check if it is installed correctly. The docs only explain how to run the bear by specifying it directory.